geometry.pyi

```diff
@@ -180,4 +180,8 @@ class Polygon:
     @overload
     def __init__(self, polygon: Polygon) -> None: ...
     def __copy__(self) -> "Polygon": ...
+    @staticmethod
+    def normal_polygon(
+        sides: int, center: Sequence[float], radius: float, angle: float = 0
```

Comment:
HEre we should add a `, /` to signify all parameters are positional
```suggestion
        sides: int, center: Sequence[float], radius: float, angle: float = 0, /
```

Reply:
python 3.8 feature so no